org.elastic.jdt.ls.product/all.policy

```diff
@@ -0,0 +1,3 @@
+grant {
```

Comment:
Any document updates on the design of this?

Reply:
Will do